### ISSUE TITLE
--geometry retro-compatibility broken between yad 0.40.0 <-> 10.90

### ISSUE THREAD

**Issue by sputnick-dev** [v1cont/yad#228]
Used yad 0.40.0 (GTK+ 3.24.33) on Linux Mint 21 vanessa

Now, using latest `AppImage`:

```
yad --version 2>/dev/null
10.90 (GTK+ 3.24.30)
```

The command that was putting the box on the top right before:

    yad --no-focus --on-top --geometry -0+80 --question --title 'title'  --text  "text"

Now with recent version, the box appears in top left.

Tried many ways to change `--geometry` at the right place, but fails

**Comment by v1cont** [comment]
yes, it's a bug. as a quick and dirty workaround you may use -1+80
